pkg/net/libp2p/key_test.go

```diff
@@ -84,14 +98,17 @@ func TestOperatorPublicKeyToNetworkPublicKey(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if !reflect.DeepEqual(networkPublicKey.Curve.Params(), DefaultCurve.Params()) {
-		t.Errorf("network public key uses the wrong curve")
+	secp256k1PublicKey, ok := networkPublicKey.(*libp2pcrypto.Secp256k1PublicKey)
+	if !ok {
+		t.Fatal("wrong type of public key")
 	}
```

Comment:
Let's change this in the same way as proposed above

Reply:
Done in https://github.com/keep-network/keep-core/commit/027febb4837f04ec11bc4e208c0b1e28a8da3642.